rotkehlchen/chain/evm/decoding/airdrops.py

```diff
@@ -0,0 +1,60 @@
+from typing import TYPE_CHECKING, Optional
+
+from rotkehlchen.accounting.structures.types import HistoryEventSubType, HistoryEventType
+from rotkehlchen.chain.ethereum.modules.airdrops.constants import CPT_COW
+from rotkehlchen.chain.ethereum.utils import asset_normalized_value
+from rotkehlchen.chain.evm.decoding.structures import (
+    DEFAULT_DECODING_OUTPUT,
+    DecoderContext,
+    DecodingOutput,
+)
+from rotkehlchen.utils.misc import hex_or_bytes_to_address, hex_or_bytes_to_int
+
+if TYPE_CHECKING:
+    from rotkehlchen.accounting.structures.evm_event import EvmEvent
+    from rotkehlchen.assets.asset import Asset, CryptoAsset
+    from rotkehlchen.fval import FVal
+    from rotkehlchen.types import ChecksumEvmAddress
+
+COW_PROTOCOL_CLAIMED = b'\xd46\xe9\x97=\x1eD\xd4\r\xb4\xd4\x11\x9e<w<\xad\xb12;&9\x81\x96\x8c\x14\xd3\xd1\x91\xc0\xe1H'  # noqa: E501
+
+
+def make_claim(
+        event: 'EvmEvent',
+        user_address: 'ChecksumEvmAddress',
+        amount: 'FVal',
+        asset: 'Asset',
+        counterparty: str,
+        notes: Optional[str],
```

Comment:
you already made it optional so if optional generate it using the template and if provided you can overwrite it